### ISSUE TITLE
[$30] Some handle colors are not correct

### ISSUE THREAD

**Issue by sdgun** [topcoder-platform/forums#109]
**Steps**
Go to the forums as a normal user with below handle colors, add forum posts and check the handle color
- Gray (ex user - sdguntcqa)
- Green (ex user - pvmagacho)
- Orange  (ex user - tonyj)

**Result**
Production - Vanilla forum
Gray - purple(#7a21b5)
Green(#00a95f) -  #00A900
Orange(#ff5100) - #fc9900

Note: We don't have users to check the colors Red, Blue and Black. Please check the used colors for them too

Ex - Gray 
------
![image](https://user-images.githubusercontent.com/4592712/98197364-9011e700-1f4c-11eb-8ed5-6cdca6a8b968.png)

![image](https://user-images.githubusercontent.com/4592712/98197423-af107900-1f4c-11eb-9eec-0d69b0dde4dc.png)

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30148998 has been created for this ticket.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by jmgasper** [comment]
@atelomycterus - Can you double check that the colours are consistent with the main site please?

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30148998 has been updated - it has been assigned to obog.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by atelomycterus** [comment]
@jmgasper Could you provide a refence to code which deployed in prod? So I can verify rating and all colors. 

Topcoder Color plugin uses old colors that are slightly different from prod.

**Comment by jmgasper** [comment]
@atelomycterus - I've asked Topcoder admins for some help here.

**Comment by jmgasper** [comment]
@atelomycterus - Stole this out of the arena code:

![Key](https://user-images.githubusercontent.com/21790/98301840-8ac9a080-200f-11eb-8069-c7996cb655ba.jpg)

```
.rating-red,
.rating-red:hover{
    color: #ee0000;
}
.rating-yellow,
.rating-yellow:hover{
    color: #ddcc00;
}
.rating-blue,
.rating-blue:hover{
    color: #6666ff;
}
.rating-green,
.rating-green:hover{
    color: #00a900;
}
.rating-grey,
.rating-grey:hover{
    color: #999;
}
.rating-none,
.rating-none:hover{
    color: #000;
}
.rating-admin,
.rating-admin:hover{
    color: #ff9900;
}
```

**Comment by atelomycterus** [comment]
@jmgasper Thanks! Please clear browser cache before testing.

**Comment by sdgun** [comment]
Verified with the available users with rating colors yellow, orange, green and black.

**Comment by jmgasper** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30148998<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by sdgun** [comment]
@jmgasper This issue can be seen again in Dev, the handle name is linked to Production profile page. The handle colors do not match with production handle color or Dev colors.

![image](https://user-images.githubusercontent.com/4592712/99040039-6d1bae80-25ae-11eb-8c63-6e223dec06a0.png)

**Comment by jmgasper** [comment]
@atelomycterus - Did we break something here?  I don't see anything obvious in the latest changes.

**Comment by atelomycterus** [comment]
@jmgasper I see the error:
![image](https://user-images.githubusercontent.com/10196632/99145887-92c6b780-2683-11eb-8fdb-9555a44eaff3.png)

We use Topcoder Roles API to check admin roles. It was implemented before synchronization of Topcoder Roles  
I've already checked all values. One parameter (`AUTH0_AUDIENCE`) has been changed. I think the current value is for prod, not dev. The previous value was 'https://m2m.topcoder-dev.com/'. 
 
Maybe we can't stop using the Topocoder role API.
If we use vanilla role information it may not be up to date.
For example, a user has a role added or removed. The user hasn’t signed in to Vanilla yet, 
or new groups / chellenge haven’t been created yet where he participates.
